internal/command/views/json/query.go

```diff
@@ -34,15 +36,17 @@ func NewQueryStart(addr addrs.AbsResourceInstance, input_config cty.Value) Query
 	}
 }
 
-func NewQueryResult(addr addrs.AbsResourceInstance, value cty.Value) QueryResult {
-	return QueryResult{
-		Address:        addr.String(),
+func NewQueryResult(listAddr addrs.AbsResourceInstance, value cty.Value, generated *genconfig.Resource) QueryResult {
+	result := QueryResult{
+		Address:        generated.Addr.String(),
 		DisplayName:    value.GetAttr("display_name").AsString(),
 		Identity:       marshalValues(value.GetAttr("identity")),
-		ResourceType:   addr.Resource.Resource.Type,
+		ResourceType:   listAddr.Resource.Resource.Type,
 		ResourceObject: marshalValues(value.GetAttr("state")),
-		// TODO: Add config once we have it available
+		Config:         string(generated.Body),
```

Comment:
I think this should be `generated.String()` to output the full resource block with labels